packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -6,6 +6,7 @@
  */
 import {
     APIFeature,
+    APIVersion,
```

Comment:
We should be using [isAPIFeatureEnabled](https://github.com/salesforce/lwc/blob/94432b279559412effb29e3c1b9d7fb1ead8742d/packages/%40lwc/shared/src/api-version.ts#L77) and its abstractions rather than directly checking the version here.